source/server/configuration_impl.h

```diff
@@ -101,7 +101,10 @@ class MainImpl : Logger::Loggable<Logger::Id::config>, public Main {
   Upstream::ClusterManager* clusterManager() override { return cluster_manager_.get(); }
   std::list<Stats::SinkPtr>& statsSinks() override { return stats_sinks_; }
   std::chrono::milliseconds statsFlushInterval() const override { return stats_flush_interval_; }
-  const Watchdog& watchdogConfig() const override { return *watchdog_; }
+
+  const Watchdog& mainThreadWatchdogConfig() const override { return *main_thread_watchdog_; }
+
+  const Watchdog& workerWatchdogConfig() const override { return *worker_watchdog_; }
```

Comment:
nit: The newlines between the accessor methods is not required, consider removing them for consistency with other accessors above.

Reply:
Done.